tests/unit/test_tablets.py

```diff
@@ -0,0 +1,85 @@
+        self.assertEqual(tablets_list[2].first_token, -3611686018427387905)
+        self.assertEqual(tablets_list[2].last_token, -6)
+        self.assertEqual(tablets_list[3].first_token, -1)
+        self.assertEqual(tablets_list[3].last_token, 2305843009213693951)
+
+        tablets.add_tablet("test_ks", "test_tb", Tablet(3305843009213693951, 4305843009213693951, None))
+        tablets_list = tablets._tablets.get(("test_ks", "test_tb"))
+        self.assertEqual(len(tablets_list), 5)
+        self.assertEqual(tablets_list[0].first_token, -8611686018427387905)
+        self.assertEqual(tablets_list[0].last_token, -7917529027641081857)
+        self.assertEqual(tablets_list[1].first_token, -6917529027641081857)
+        self.assertEqual(tablets_list[1].last_token, -4611686018427387905)
+        self.assertEqual(tablets_list[2].first_token, -3611686018427387905)
+        self.assertEqual(tablets_list[2].last_token, -6)
+        self.assertEqual(tablets_list[3].first_token, -1)
+        self.assertEqual(tablets_list[3].last_token, 2305843009213693951)
+        self.assertEqual(tablets_list[4].first_token, 3305843009213693951)
+        self.assertEqual(tablets_list[4].last_token, 4305843009213693951)
+
+        tablets.add_tablet("test_ks", "test_tb", Tablet(-8011686018427387905, -7987529027641081857, None))
+        tablets_list = tablets._tablets.get(("test_ks", "test_tb"))
+        self.assertEqual(len(tablets_list), 5)
+        self.assertEqual(tablets_list[0].first_token, -8011686018427387905)
+        self.assertEqual(tablets_list[0].last_token, -7987529027641081857)
+        self.assertEqual(tablets_list[1].first_token, -6917529027641081857)
+        self.assertEqual(tablets_list[1].last_token, -4611686018427387905)
+        self.assertEqual(tablets_list[2].first_token, -3611686018427387905)
+        self.assertEqual(tablets_list[2].last_token, -6)
+        self.assertEqual(tablets_list[3].first_token, -1)
+        self.assertEqual(tablets_list[3].last_token, 2305843009213693951)
+        self.assertEqual(tablets_list[4].first_token, 3305843009213693951)
+        self.assertEqual(tablets_list[4].last_token, 4305843009213693951)
+
+        tablets.add_tablet("test_ks", "test_tb", Tablet(-9011686018427387905, -3611686018427387995, None))
+        tablets_list = tablets._tablets.get(("test_ks", "test_tb"))
+        self.assertEqual(len(tablets_list), 4)
+        self.assertEqual(tablets_list[0].first_token, -9011686018427387905)
+        self.assertEqual(tablets_list[0].last_token, -3611686018427387995)
+        self.assertEqual(tablets_list[1].first_token, -3611686018427387905)
+        self.assertEqual(tablets_list[1].last_token, -6)
+        self.assertEqual(tablets_list[2].first_token, -1)
+        self.assertEqual(tablets_list[2].last_token, 2305843009213693951)
+        self.assertEqual(tablets_list[3].first_token, 3305843009213693951)
+        self.assertEqual(tablets_list[3].last_token, 4305843009213693951)
```

Comment:
If at one point this test fails, it may not be immediately obvious to the person fixing this what each of those blocks is supposed to check (and consequently, what doesn't work in the code).
It's a good practice to keep test functions small so that they check one thing, and explain exactly what they check - either in the name or in the comment. Here, each of the blocks could be it's own function.
Apart from making those tests easier to understand, it may also be easier to find untested edge cases - currently I don't know what you test and what you don't without carefully analyzing all the numbers in the code. 
It's also fine to write helper functions to improve readability - you could create a function that takes a list of tablets and a list of pairs (ranges) and checks that they are equal - you wouldn't have to write that many asserts. Then you could use `message` argument to `assertEqual` so that in case of failing test content of tablets is printed, resulting in easier debugging. That would probably require implementing `__str__` / `__repr__` (not sure which of them, preferably both) on `Tablet` class - which is imo a good idea even by itself and will be useful in the future.

Look e.g. at:
https://github.com/scylladb/java-driver/blob/scylla-3.x/driver-core/src/test/java/com/datastax/driver/core/
Tests in Rust driver (search for `#[tokio::test]` and `#[test]`)
https://github.com/scylladb/scylladb/tree/master/test/topology